subsys/bluetooth/shell/bt.c

```diff
@@ -3480,6 +3485,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(bt_scan_filter_clear_cmds,
 SHELL_STATIC_SUBCMD_SET_CREATE(bt_cmds,
 	SHELL_CMD_ARG(init, NULL, "[no-settings-load], [sync]",
 		      cmd_init, 1, 2),
+	SHELL_CMD_ARG(disable, NULL, HELP_NONE, cmd_disable, 1, 2),
```

Comment:
There are no optional arguments to the disable command.
```suggestion
	SHELL_CMD_ARG(disable, NULL, HELP_NONE, cmd_disable, 1, 0),
```